### PR TITLE
Emit 'page-change' event with the updated page number if itemsPerPage or totalItems changes.

### PR DESCRIPTION
**Pull request by pavelav**
This should take care of the issue https://github.com/arnedesmedt/vue-ads-pagination/issues/16

### CODE REVIEW

File: src/components/Pagination.vue [hunk 1]
```diff
@@ -185,7 +185,12 @@ export default {
         },
 
         rangeChange () {
-            this.$emit('range-change', this.start, this.end);
+            if (this.page >= this.totalPages && this.totalPages !== 0) {
+                this.$emit('page-change', this.totalPages - 1);
+            }
+            else {
+                this.$emit('range-change', this.start, this.end);
```
Comment (pavelav) [inline src/components/Pagination.vue]: This change assumes that the parent component will update `page` property in response to `'page-change'` event (that's how the page buttons work). As a result `pageChange` will be called that calls `rangeChange` that now falls into the `else` branch.

**Comment by pavelav** [comment]
Also would like to bring your attention to this PR. The changes are simple, but solve an issue. Thank you!

**Comment by arnedesmedt** [comment]
Sorry, Didn't see this one either, Thx for the PR!

**Comment by arnedesmedt** [comment]
A patch is released:  version 2.1.7

**Comment by pavelav** [comment]
> A patch is released: version 2.1.7

Thanks! Please update vue-ads-table-tree to require it.

**Comment by arnedesmedt** [comment]
Good point, forgot that.

**Comment by arnedesmedt** [comment]
version 2.4.1 is available.

**Pull request merged**